go-selinux/selinux_linux.go

```diff
@@ -79,8 +77,20 @@ var (
 	// for attrPath()
 	attrPathOnce   sync.Once
 	haveThreadSelf bool
+
+	// for getPolicyRoot
+	getPolicyRootOnce sync.Once
+	policyRoot        string
 )
 
+func getPolicyRoot() string {
+	getPolicyRootOnce.Do(func() {
+		policyRoot = filepath.Join(selinuxDir, readConfig(selinuxTypeTag))
```

Comment:
I noticed that `readConfig` is also called in `defaultEnforceMode()`. Wondering if instead of applying a `sync.Once` here, we should make `readConfig` store the results as a map (similar to `doReadLabels()`) and have a sync.Once for that 🤔 

Not sure if that makes sense (mostly thinking out loud as usual)

Reply:
I thought about it as well (first time a year or so ago when I saw the code for the first time). This is a small text file, only containing two variables, and `DefaultEnforceMode` is (I guess) rarely used, so it's not worth optimizing for.